.gitignore

```diff
@@ -59,3 +59,6 @@ typings/
 
 
 \.idea/
+
+# osx files
+.DS_S*
```

Comment:
is there anything else than `.DS_Store` to ignore?

Reply:
If anyone uses vscode... maybe

Reply:
> If anyone uses vscode... maybe

😂

Reply:
> is there anything else than `.DS_Store` to ignore?

Don't think so

Reply:
Can we then just be specific about `.DS_Store` ? I think is good to be specific in gitignore

Reply:
Probably yes, but not sure if its possible to find sometimes the same file with different case.

> Can we then just be specific about `.DS_Store` ? I think is good to be specific in gitignore